support-e2e/tests/test/ticketTailor.ts

```diff
@@ -0,0 +1,58 @@
+import { expect, test } from '@playwright/test';
+import { email, firstName, lastName } from '../utils/users';
+import { setupPage } from '../utils/page';
+
+type TestDetails = {
+	internationalisationId: string;
+	eventId: string;
+};
+
+export const testTicketTailor = (testDetails: TestDetails) => {
+	const { internationalisationId, eventId } = testDetails;
+
+	test(`${internationalisationId.toUpperCase()} event ${eventId}`, async ({
+		context,
+		baseURL,
+	}) => {
+		const url = `/${internationalisationId}/events/${eventId}`;
+		const page = await context.newPage();
+		await setupPage(page, context, baseURL, url);
+
+		// -------- iFrame Locator Notes (wip) ---------
+		// page.frameLocator(
+		//    "iframe"
+		//    "iframe[name^/title^='']"
+		//    "iframe[url^='https://www.tickettailor.com/events/guardianlivecode/1354460/book']" -> ? Possible ?
+		//  )
+		//  .first() / .nth(*) / .last()
+		//  .locator(
+		//      'body.dom-ready' => ? Possible, class gets added to the iframe body after the JavaScript has finished executing ?
+		//      '[role="heading"]:has-text["CODE"]' Or .getByRole('heading', { name: `CODE` }) Or .getByText('CODE')
+		//      '[role="link"]:has-text["Next"]' Or .getByRole('link', { name: `Next` })
+		//      '#<Name>'
+
+		const iFrame = page.frameLocator('iframe').first();
+		await expect(
+			iFrame.locator('[role="heading"]:has-text["CODE"]'),
+		).toBeVisible({
+			timeout: 10000,
+		});
```

Comment:
@paul-daniel-dempsey I had a quick go here - and it seems you need to access the [contentFrame](https://playwright.dev/docs/api/class-elementhandle#element-handle-content-frame).